flake.nix

```diff
@@ -0,0 +1,41 @@
+{
+  description = "Hardware and Firmware of the Solo Quadruped Master Board.";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs?ref=nixos-unstable";
+  };
+
+  outputs =
+    { self, nixpkgs }:
+    let
+      pkgs = nixpkgs.legacyPackages.x86_64-linux;
+    in
+    {
+      packages.x86_64-linux.default = pkgs.stdenv.mkDerivation {
+
+        pname = "odri-masterboard-sdk";
+        version = "1.0.7";
+
+        src = builtins.path {
+          name = "sdk";
+          path = ./sdk/master_board_sdk;
+        };
+
+        doCheck = true;
+
+        nativeBuildInputs = with pkgs; [
+          jrl-cmakemodules
+          cmake
+          python312
+          python312Packages.boost
```

Comment:
Hum, do we need that package both in native and propagated ?

Reply:
woops, i'll blame too much caffeine on this one